spec/acceptance/version.rb

```diff
@@ -32,20 +32,20 @@
   $vhost            = '/etc/apache2/sites-available/15-default.conf'
   $vhost_dir        = '/etc/apache2/sites-enabled'
   $run_dir          = '/var/run/apache2'
-  $doc_root          = '/var/www'
+  $doc_root = '/var/www'
```

Comment:
Can you add in more spacing here to match the other equals signs above and below.